packages/app/obojobo-repository/shared/components/pages/page-dashboard-client.jsx

```diff
@@ -1,8 +1,10 @@
-import { hydrateEl } from '../../react-utils'
+import { hydrateEl, hydrateElWithoutStore } from '../../react-utils'
 import Dashboard from '../dashboard-hoc'
 import DashboardReducer from '../../reducers/dashboard-reducer'
+import Notification from '../notification.jsx'
```

Comment:
Moving the notification component into the Dashboard and rendering it exclusively on the client side will remove the need for server-side rendering and hydration, so this import can go.